code/modules/mob/language/languages.dm

```diff
@@ -57,6 +57,17 @@
 
 	syllables = list("die", "das", "wein", "mir", "und", "wir", "ein", "nein", "gen", "en", "sauen", "bin", "nein", "rhein", "deut", "der", "lieb", "en", "stein", "nein", "ja", "wolle", "sil", "bei", "der", "sie", "sch", "kein", "nur", "ach", "kann", "volk", "vau", "gelb", "grun", "macht", "zwei", "vier", "nacht", "tag")
 
+/datum/language/scandinavian
+	name = LANGUAGE_SCANDINAVIAN
+	desc = "While not technically one language, Scandinavian languages have grown similar and are nearly indistinguishable from one another unless you actually know the languages."
+	speech_verb = "blubbers"
+	ask_verb = "queries"
+	exclaim_verb = "yelps"
+	color = "blue"
```

Comment:
Use a hexcode for color.
And if you want to keep it similar to how the other languages do color, add the color ID and hexcode for it here.
I say ad it, dont know TGUI myself well enough to know if its needed or not.

cmss13\tgui\packages\tgui-panel\styles\goon\chat-dark.scss